### ISSUE TITLE
Default voice config is not applied

### ISSUE THREAD

**Issue by bklang** [adhearsion/adhearsion#472]
I'm not sure if this is an Adhearsion problem, a mod_rayo problem, or a mod_unimrcp problem. 

Setting `config.platform.media.default_voice` results in the value being correctly set on the `<output/>` component, but the resulting voice is still the platform default.

Trace log:

```
    <output xmlns="urn:xmpp:rayo:output:1" voice="chloe" renderer="unimrcp">
      <document content-type="application/ssml+xml">&lt;speak xmlns="http://www.w3.org/2001/10/synthesis" version="1.0" xml:lang="en-US"&gt;You answered incorrectly. Press 1 to hear hold music. Press 2 if you are tired of waiting on hold. Press 3 to hear a joke. &lt;/speak&gt;</document>
    </output>
```

But the voice heard was not Chloe.
